CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits in the store-gateway. #4056
 * [CHANGE] Packaging: Migrate FPM packaging solution to NFPM. Rationalize packages dependencies and add package for all binaries. #3911
+* [FEATURE] Query-frontend: Introduce experimental `-query-frontend.max-series-per-shard` to allow query sharding to take into account cardinality of similar requests executed previously. #4121
```

Comment:
Please add this feature to the list of experimental features listed in `docs/sources/mimir/operators-guide/configure/about-versioning.md`